### ISSUE TITLE
license?

### ISSUE THREAD

**Issue by caseyjlaw** [demorest/sdmpy#13]
I am seeing this code getting more useful to a wider audience (custom SDM generation, realfast data products, python 3 support). You may want to add a license to the repo so users know what they can do with the code. I'm guessing the NRAO also has some obligations...

**Comment by demorest** [comment]
Yes this is a good idea.  I will investigate a bit and find out what is appropriate to use in this case.
